test_project/settings.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+import dj_database_url
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
I'm not sure why `flake8` is not triggering an error at this position. After the imports it needs two blank lines.